src/components/EditorTopBar.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { AppServicesProvider } from "./AppServicesContext";
+import { EditorTopBar } from "./EditorTopBar";
+
+const appConfiguration = {
+  dopplerExternalUrls: {
+    home: "homeUrl",
+    campaigns: "campaignsUrl",
+    lists: "listsUrl",
+    controlPanel: "controlPanelUrl",
+  }
+};
+
+describe(EditorTopBar.name, () => {
+  it("should render Exit Button", async () => {
+    // Arrange
+    const appServices = { appConfiguration } as any;
+
+    // Act
+    render(
+      <AppServicesProvider
+      appServices={appServices}
+    >
+      <EditorTopBar onSave={() => {}}></EditorTopBar>
+      </AppServicesProvider>
+    );
+
+    // Assert
+    screen.getByText('Salir del Editor');
+  });
+
+  it("should render Exit Options after click on Exit Button", async () => {
+    // Arrange
+    const appServices = { appConfiguration } as any;
+
+    // Act
+    render(
+      <AppServicesProvider
+      appServices={appServices}
+    >
+      <EditorTopBar onSave={() => {}}></EditorTopBar>
+      </AppServicesProvider>
+    );
+
+    userEvent.click(screen.getByText('Salir del Editor'));
+
+    // Assert
+    expect(screen.queryByText('Inicio')).not.toBeNull;
```

Comment:
Entiendo que el `getByText` es suficiente para la aserción.

```suggestion
    // Assert
    screen.getByText('Inicio');
```

Reply:
Fixed